public/docs/ts/latest/cookbook/production-deploy.jade

```diff
@@ -0,0 +1,316 @@
+
+.l-main-section
+:marked
+  ## Simplest deploy possible
+
+  The simplest possible way to deploy your app is to use your development environment.
+  It is already working locally after all, so it should work live.
+
+  1. Copy over your local project folder to your server.
+  2. In your server, edit `index.html` to have the right `<base href="/">` tag. If you are
+  using a subfolder (e.g. `www.mysite.com/myapp/`) it should be `/myapp/` otherwise leave it as `/`.
+  [More on this later](#angular-configuration).
+  3. Configure your server to redirect requests for missing files to `index.html` instead.
+  [More on this later](#server-configuration).
+
+  And this is all you need to publish your app!
+
+  It's not a very good production deploy though. 
+  There's more you can do to make our site fast for users.
+ 
+.l-main-section
+:marked
+  ## Minimizing your payload
+
+  One of the easiest optimizations you can do is to reduce the code users have to download.
+  
+  There are several techniques for achieving a smaller payload for your app:
+
+  - Bundling: concatenating several modules into a single file (bundle). 
+  - Inlining: add html and css as strings inside components.
+  - Minification: removing all unnecessary whitespace and optional tokens.
+  - Uglification: rewriting code to use shorter variable/function names.
+  - Dead Code Elimination: removing code/variables that will never used.
+  - Tree shaking: removing unused module exports.
+  - Ahead-of-Time (AoT) Compilation: pre-compiling Angular templates.
+
+  Each of these do different things overall, but they all work together and their effects
+  compound on each other.
+
+  ### Less files
+  Bundling and inlining reduce your app's load time smaller by serving less files and thus
+  less round trips. 
+  For small files, the overwhelming majority of time it takes to get it is not spent downloading,
+  but rather communicating with the server and coordinating the transfer. 
+  You can read more about resource timings in the 
+  [Chrome DevTools Network Performance page](https://developers.google.com/web/tools/chrome-devtools/network-performance/understanding-resource-timing)   
+
+  ### Smaller files
+  Minification, Uglification and Dead Code Elimination all contribute to reduce the total size
+  of your code to the smallest possible amount of bytes, which will result in a smaller
+  transfer time.
+
+  Tree shaking also reduces the total size but does so by removing whole exports from modules.
+  It was popularized by [Rollup](http://rollupjs.org/) and you can read more about it in 
+  [this post](https://medium.com/@Rich_Harris/tree-shaking-versus-dead-code-elimination-d3765df85c80#.15ih9cyvl).
+
+  ### Template compilation
+  AoT compilation reduces render time by pre-compiling Angular templates that otherwise would be
+  compiled at runtime. 
+  It also reduces payload size since the Angular compiler itself is no longer needed, and nor are
```

Comment:
Consider clarifying "is no longer needed". Maybe something like: "does not need to be downloaded to the client"  (or user)